Formula/t/thors-serializer.rb

```diff
@@ -25,6 +25,8 @@ class ThorsSerializer < Formula
 
   def install
     ENV["COV"] = "gcov"
+    mkdir -p ~/.ssh
+    ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```

Comment:
Why is this required?

Reply:
The first run failed with:

> Host key verification failed.

When it was pulling down the submodules. I was trying to solve this issue.
I have removed it.